src/shims/shims.proj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="Directory.Build.props" />
 
+  <UsingTask TaskName="GenFacadesTask" AssemblyFile="$(PackagesDir)/$(MicrosoftDotNetGenFacadesPackage.ToLower())/$(MicrosoftDotNetGenFacadesPackageVersion)/tools/netcoreapp2.1/Microsoft.DotNet.GenFacades.dll" />
```

Comment:
Is this necessary? now that you are importing the targets in Directory.Build.targets?